packages/cbioportal-frontend-commons/src/lib/StringUtils.ts

```diff
@@ -72,3 +72,17 @@ export function isUrl(str: string) {
     const pattern = /^http[s]?:\/\//;
     return pattern.test(str);
 }
+
+export const LESS_THAN_ENTITY = '&lt;';
+export const GREATER_THAN_ENTITY = '&gt;';
+
+export function trimOffHtmlTagsByEntity(htmlStr: string) {
```

Comment:
Should we just use [this package](https://www.npmjs.com/package/string-strip-html) instead of implementing our own custom function?

Reply:
@onursumer I thought about that, but we also need to decode it first before using this package. Thought that would be overkill to include two new packages to the project.

Reply:
[Here](https://www.npmjs.com/package/string-strip-html#features) it says: 

```
Uses recursive HTML decoding, so there's no way to cheat this library by using any kind of HTML encoding (unless you turn the decoding off via opts.skipHtmlDecoding)
```

This sounds like it actually handles encoded strings by default, so maybe we don't need any other package.

Reply:
Great find! Let me give it a try.

Reply:
Works really well, also passed my tests.
Gonna remove all my script and tests.